src/mono/netcore/System.Private.CoreLib/src/System/Threading/ThreadPool.Browser.Mono.cs

```diff
@@ -4,34 +4,40 @@
 using System.Diagnostics;
 using System.Collections.Generic;
 using System.Runtime.CompilerServices;
-using System.Runtime.Versioning;
 using System.Diagnostics.CodeAnalysis;
 using Microsoft.Win32.SafeHandles;
 
 namespace System.Threading
 {
-    [UnsupportedOSPlatform("browser")]
```

Comment:
This needs to come back to align with ref version

Reply:
I had changed `RegisteredWaitHandle` to be a partial class and kept the attribute in the shared implementation in `ThreadPool.cs`